channel_setup/delete_prereqs.sh

```diff
@@ -1,3 +1,22 @@
 #!/bin/bash
+
+
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "NAME"
+  echo "  delete_prereqs - Delete channel prerequisites"
+  echo
+  echo "DESCRIPTION"
+  echo "  Deletes all numeracy channel prerequisites"
```

Comment:
It's better to refer to them as content prerequisites instead of channel prerequisites